sentinelsat/sentinel.py

```diff
@@ -939,7 +939,40 @@ def check_bounds(geometry):
     wkt = re.sub(r'(?<!\d) ', '', wkt)
     return wkt
 
+def naturalname_to_wkt(naturalname):
```

Comment:
I prefer `placename` instead of `naturalname`. This would also mirror the nominatim key `place` which will probably be the one the query hits most often.

Reply:
ok!